code/modules/food_and_drinks/food/foods/misc_food.dm

```diff
@@ -63,33 +63,101 @@
 //		Salads		//
 //////////////////////
 
-/obj/item/reagent_containers/food/snacks/aesirsalad
-	name = "aesir salad"
-	desc = "Probably too incredible for mortal men to fully enjoy."
-	icon_state = "aesirsalad"
+/obj/item/reagent_containers/food/snacks/salad
+	name = "salad"
+	desc = "Should not spawn or be possible to be created. Contact a coder if this somehow exists."
+	icon = 'icons/obj/food/soupsalad.dmi'
+	icon_state = "salad"
 	trash = /obj/item/trash/snack_bowl
 	filling_color = "#468C00"
 	bitesize = 3
+	list_reagents = list("nutriment" = 6,"vitamin" = 4)
+	tastes = list("lettuce" = 2)
+
+/obj/item/reagent_containers/food/snacks/salad/aesir
+	name = "aesir salad"
+	desc = "Probably too incredible for mortal men to fully enjoy."
+	icon_state = "aesirsalad"
 	list_reagents = list("nutriment" = 8, "omnizine" = 8, "vitamin" = 6)
 	tastes = list("divinity" = 1, "lettuce" = 1)
 
-/obj/item/reagent_containers/food/snacks/herbsalad
+/obj/item/reagent_containers/food/snacks/salad/antipasto
+	name = "antipasto salad"
+	desc = "A traditional Italian salad made of salami, mozzarella cheese, olives, and tomatoes. Often served as a first course meal."
+	icon_state = "antipasto_salad"
+	list_reagents = list("nutriment" = 12, "protein" = 6, "vitamin" = 6)
+	tastes = list("lettuce" = 2, "salami" = 2, "mozzarella cheese" = 2, "tomatoes" = 2, "dressing" = 1)
+
+/obj/item/reagent_containers/food/snacks/salad/caesar
+	name = "Caesar salad"
+	desc = "A simple yet flavorful salad of onions, lettuce, croutons, and shreds of cheese dressed in oil. Comes with a slice of pita bread!"
+	icon_state = "caesar_salad"
+	list_reagents = list("nutriment" = 12, "vitamin" = 6)
+	tastes = list("healthy greens" = 2, "olive dressing" = 2, "feta cheese" = 2, "pita bread" = 1)
+
+/obj/item/reagent_containers/food/snacks/salad/fruit
+	name = "fruit salad"
+	desc = "Your standard fruit salad."
+	icon_state = "fruitsalad"
+	list_reagents = list("nutriment" = 9, "vitamin" = 4)
+	tastes = list("fruit" = 1)
```

Comment:
Same here, it doesn't hurt much, but I don't think this needs to be a weighted list